content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -1,165 +1,320 @@
+   "title": "Comparing Database Types: How Database Types Evolved to Meet Different Needs",
+   "authorId": 5,
+   "author": {
+     "id": 5,
+     "email": "jane@prisma.io",
+     "name": "Jane"
+   }
+   },
+   {
+   "id": 12,
+   "content": "https://quantifiedself.com/get-started/",
+   "title": "Analysing Sleep Patterns: The Quantified Self",
+   "authorId": 5,
+   "author": {
+     "id": 5,
+     "email": "jane@prisma.io",
+     "name": "Jane"
+   }
+   },
+   {
+   "id": 13,
+   "content": "",
+   "title": "Requiem for String Orchestra",
+   "authorId": 6,
+   "author": {
+     "id": 6,
+     "email": "toru@prisma.io",
+     "name": "Toru Takemitsu"
+   }
+   },
+   {
+   "id": 14,
+   "content": "",
+   "title": "Music of Tree",
+   "authorId": 6,
+   "author": {
+     "id": 6,
+     "email": "toru@prisma.io",
+     "name": "Toru Takemitsu"
+   }
+   },
+   {
+   "id": 15,
+   "content": "",
+   "title": "Waves for clarinet, horn, two trombones and bass drum ",
+   "authorId": 6,
+   "author": {
+     "id": 6,
+     "email": "toru@prisma.io",
+     "name": "Toru Takemitsu"
+   }
+   }
+   ]
+   ```
+
+   <Admonition type="info">
+
+   This API call uses Prisma to get all `posts` in the database with their related `authors`. The code is in the `pages/api/posts.js` file.
+
+   </Admonition>
+
+### Result
+
+You have now tested the API for your application and added data to your database.
+
+## 3. (Optional) Add a separate database for preview deployments
+
+Your application currently has a single _production_ environment, associated with the `main` Git branch of your repository. If you open a pull request to change your application, Vercel creates a new _preview_ environment.
+
+By default, Vercel uses the `DATABASE_URL` environment variable that you defined when you imported the project for both the production and preview environments. This causes problems if you create a pull request with a database schema migration, because the pull request will change the schema of the production database.
+
+In this section you associate a second PostgreSQL database with a Vercel preview environment.
+
+### Steps
+
+1. Click the **Settings** tab of your Vercel project.
+
+2. Click **Environment variables**.
+
+3. Add an environment variable with a key of `DATABASE_URL` and select only the **Preview** environment option:
+
+   ![Add an environment variable for the preview environment](images/300-60-deploy-to-vercel-preview-environment-variable.png)
+
+4. Set the value to the connection string of your second PostgreSQL database. As before, the connection string has the following format:
+
+   ```code
+   postgresql://__USER__:__PASSWORD__@__HOST__/__DATABASE__
+   ```
+
+5. Click **Save**.
+
+### Result
+
+You should see the `DATABASE_URL` environment variable for the preview environment in your list of environment variables:
+
+![List of environment variables](images/300-70-deploy-to-vercel-environment-variables.png)
```

Comment:
I think after this the user will need a bit more explanation how this now works and what happens.

"Now your preview deployments will use their own `DATABASE_URL` to apply migrations, so you can properly test them in a preview environment before you merge your PR with the migration to your main branch which will then also update your production database." or simiar.